src/platform/forms-system/src/js/review/ReviewCollapsibleChapter.jsx

```diff
@@ -83,208 +83,253 @@ class ReviewCollapsibleChapter extends React.Component {
-        <div
-          className="usa-accordion-content schemaform-chapter-accordion-content"
-          aria-hidden="false"
-        >
-          {ChapterDescription && (
-            <ChapterDescription
-              viewedPages={viewedPages}
-              pageKeys={pageKeys}
+    const updateButton = (
+      <ProgressButton
+        submitButton
+        onButtonClick={() => {
+          // recheck _all_ validations after the user clicks the
+          // update page button - needed to dynamically update
+          // accordion headers
+          this.checkValidation();
+          focusOnChange(
+            `${page.pageKey}${
+              typeof page.index === 'number' ? page.index : ''
+            }`,
+          );
+        }}
+        buttonText="Update page"
+        buttonClass="usa-button-primary"
+        ariaLabel={ariaLabel}
+      />
+    );
+
+    // These are functions so React doesn't create the component if we don't use
+    // it, but we still get the brevity later when we `renderCustom ?
+    // customContent() : schemaformContent();`
+    const schemaformContent = () => (
+      <SchemaForm
+        name={page.pageKey}
+        title={title}
+        data={pageData}
+        appStateData={page.appStateData}
+        schema={pageSchema}
+        uiSchema={pageUiSchema}
+        trackingPrefix={this.props.form.trackingPrefix}
+        hideHeaderRow={page.hideHeaderRow}
+        hideTitle={this.shouldHideExpandedPageTitle(
+          expandedPages,
+          this.getChapterTitle(chapterFormConfig),
+          title,
+        )}
+        pagePerItemIndex={page.index}
+        onBlur={this.props.onBlur}
+        onEdit={() => this.handleEdit(page.pageKey, !editing, page.index)}
+        onSubmit={({ formData }) =>
+          this.handleSubmit(formData, page.pageKey, page.arrayPath, page.index)
+        }
+        onChange={formData =>
+          this.onChange(
+            typeof page.updateFormData === 'function'
+              ? page.updateFormData(form.data, formData)
+              : formData,
+            page.arrayPath,
+            page.index,
+          )
+        }
+        uploadFile={this.props.uploadFile}
+        reviewMode={!editing}
+        formContext={formContext}
+        editModeOnReviewPage={page.editModeOnReviewPage}
+      >
+        {!editing ? <div /> : updateButton}
+      </SchemaForm>
+    );
+
+    // TODO: Add the props to CustomPage
+    const customContent = () =>
+      editing ? (
+        <page.CustomPage
+          onReviewPage
+          updatePage={({ formData }) =>
+            this.handleSubmit(
+              formData,
+              page.pageKey,
+              page.arrayPath,
+              page.index,
+            )
+          }
+        />
+      ) : (
+        this.getCustomPageContent(page, props)
+      );
+
+    const renderCustom = editing ? !!page.CustomPage : !!page.CustomPageReview;
+    const reviewContent = renderCustom ? customContent() : schemaformContent();
```

Comment:
This is new. This function (`getSchemaformPageContent`) is called when there is no `CustomPageReview`, so when in "view" mode, it renders the `SchemaForm` in review mode.

These two lines here determine what to render in the edit mode.